First/ViewController.swift

```diff
@@ -4,19 +4,20 @@
 //
 //  Created by Adam Głąbicki Airnauts on 25/07/2022.
 //
```

Comment:
jedyna rzecz do ktorej na razie moge sie przyczepic to to ze zwykle usuwamy ten komentarz na gorze :p